packages/headless/src/app/engine.ts

```diff
@@ -318,17 +314,20 @@ function shouldWarnAboutOrganizationEndpoints(
 }
 
 function shouldWarnAboutPlatformURL(options: EngineOptions<ReducersMapObject>) {
-  return !isNullOrUndefined(options.configuration.platformUrl);
+  return isNullOrUndefined(
+    options.configuration.organizationEndpoints?.platform
```

Comment:
if org end point url is defined do we care about the older platform url since it would not be used anyways?

Reply:
I think we should still warn: if an implementer set an option that's discarded/irrelevant/not needed anymore, it's still meaningful to warn and inform them.

Plus, next major, we will actually want to remove entirely the `platformUrl` option, so it also a preventive warning that it's going away, and it might actually throw error in next version.

Reply:
Makes sense 👍